packages/atomic/src/components/common/atomic-timeframe/atomic-timeframe.new.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import {renderComponent} from '@coveo/atomic/storybookUtils/render-component';
+import {wrapInSearchInterface} from '@coveo/atomic/storybookUtils/search-interface-wrapper';
+import type {Meta, StoryObj} from '@storybook/web-components';
+import {html} from 'lit/static-html.js';
+
+const {decorator, play} = wrapInSearchInterface();
+
+const meta: Meta = {
+  component: 'atomic-timeframe',
+  title: 'Atomic/TimeframeFacet/Timeframe',
+  id: 'atomic-timeframe',
```

Comment:
'atomic-timeframe-facet'
```suggestion
  id: 'atomic-timeframe-facet'
```